### PR TITLE
[JFG-949] Correct endpoint for JaaS pdf report generation

### PR DESCRIPTION
**Pull request by a-badaev**

### CODE REVIEW

File: jaas/src/main/java/com/griddynamics/jagger/jaas/service/TestEnvironmentExpirationService.java [hunk 1]
```diff
@@ -24,8 +21,11 @@ public TestEnvironmentExpirationService(TestEnvironmentDao testEnvironmentDao) {
 
     @Scheduled(fixedRateString = "${environments.cleaning.job.periodicity}")
     public void deleteExpiredEnvironmentsTask() {
-        long nowTimestampInUtc = now().withZoneSameInstant(UTC).toInstant().toEpochMilli();
-        int deleted = testEnvironmentDao.deleteExpired(nowTimestampInUtc);
-        LOGGER.info(deleted + " expired test environments deleted.");
+        int deleted = testEnvironmentDao.deleteExpired(System.currentTimeMillis());
+        if (deleted > 0) {
+            LOGGER.info("{} expired test environments has been deleted.", deleted);
+        } else {
```
Comment (anton-antonenko) [inline jaas/src/main/java/com/griddynamics/jagger/jaas/service/TestEnvironmentExpirationService.java]: Is this else needed here?

File: jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ReportingRestController.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ * JaaS REST API controller based on Spring MVC which generates Jagger test execution reports.
+ */
+@RequestMapping(value = "/report")
+@RestController
+public class ReportingRestController {
+    
+    private final DynamicReportingService dynamicReportingService;
+    
+    public ReportingRestController(DynamicReportingService dynamicReportingService) {
```
Comment (anton-antonenko) [inline jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ReportingRestController.java]: Didn't you forget to @Autowire it? Or how do you inject DynamicReportingService?

Comment (a-badaev) [inline jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ReportingRestController.java]: You are right. I forgot to @Autowire it. But it works :)
Looks like spring is smart enough to automatically autowire constructor args.

File: jaas/src/main/java/com/griddynamics/jagger/jaas/service/TestEnvironmentExpirationService.java (hunk 1 above)
Comment (a-badaev) [inline jaas/src/main/java/com/griddynamics/jagger/jaas/service/TestEnvironmentExpirationService.java]: Yes. If logs level is DEBUG and `deleted > 0` then there are would be duplicated log

Comment (anton-antonenko) [inline jaas/src/main/java/com/griddynamics/jagger/jaas/service/TestEnvironmentExpirationService.java]: Ok. I didn't notice debug)

**Comment by SokolAndrey** [comment]
+1

**Comment by anton-antonenko** [comment]
:+1:

**Pull request merged**